apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-menu/styles.scss

```diff
@@ -1,4 +1,4 @@
-@import '@automattic/onboarding/styles/mixins';
+@use '@automattic/onboarding/styles/mixins' as styles-mixins;
```

Comment:
Why was this renamed to `styles-mixins`? I think it's to avoid naming conflicts but there are no other imports in the file 🤔